.github/workflows/main.yml

```diff
@@ -13,3 +13,12 @@ jobs:
         run: |
           npm install
           make test
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Run ESLint
+        run: |
+          npm install
+          ./node_modules/.bin/eslint .
```

Comment:
@npupko what about writing make command? Or at last we can write new command in `script` section of `package.json`

Reply:
@lazycoder9 Yep, thanks.

P.S.: Today I learned what is `.PHONY`:)